outputs/costs_2050.csv

```diff
@@ -972,12 +972,12 @@ electricity grid connection,investment,148.151,EUR/kW,DEA, from old pypsa cost a
 electricity grid connection,lifetime,40.0,years,TODO, from old pypsa cost assumptions,2015.0
 electrobiofuels,C in fuel,0.9316,per unit,Stoichiometric calculation,,
 electrobiofuels,FOM,3.0,%/year,combination of BtL and electrofuels,,2015.0
-electrobiofuels,VOM,2.6044,EUR/MWh_th,combination of BtL and electrofuels,,2017.0
+electrobiofuels,VOM,2.5055,EUR/MWh_th,combination of BtL and electrofuels,,2020.0
 electrobiofuels,capture rate,0.9,per unit,Assumption based on doi:10.1016/j.biombioe.2015.01.006,,
 electrobiofuels,efficiency-biomass,1.3283,per unit,Stoichiometric calculation,,
-electrobiofuels,efficiency-hydrogen,1.2971,per unit,Stoichiometric calculation,,
-electrobiofuels,efficiency-tot,0.6563,per unit,Stoichiometric calculation,,
-electrobiofuels,investment,322224.6071,EUR/kW_th,combination of BtL and electrofuels,,2017.0
+electrobiofuels,efficiency-hydrogen,1.2176,per unit,Stoichiometric calculation,,
+electrobiofuels,efficiency-tot,0.6353,per unit,Stoichiometric calculation,,
+electrobiofuels,investment,2589.5241,EUR/kW_th,combination of BtL and electrofuels,,2020.0
```

Comment:
Do you know why these values were also affected by your changes?

Reply:
electrobiofuel values are calculated based on the parameters for efuels and BtL